Pods-ParseUI wont build with cocoapods 0.36.0.beta.2

###### Environment

- cocoapods version 0.36.0.beta.2
- xcode Version 6.1.1 (6A2008a)
- this podfile

````
platform :ios, '8.1'

pod 'JSQMessagesViewController', '~> 6.1.3'
pod 'Parse', '~> 1.6.2'
pod 'ParseUI', '~> 1.1.0'
pod 'DateTools', '~> 1.5.0'
pod 'Alamofire', '~> 1.1'
pod 'SwiftyJSON', '~> 2.1.3'
````

###### Repro

1. Create a new single page iOS app with xcode, quit xcode
2. use the `Podfile` above and `pod install`
3. Open `<Project>.xcworkspace` and build

###### Expect

1. Project builds

###### Actual

1. Get 18 linker errors about undefined symbols for architecture x86_64 while building Pods-ParseUI

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,11 +2,12 @@ language: objective-c
 before_install:
   - brew update
   - brew reinstall xctool
-  - gem install cocoapods --quiet
+  - gem install cocoapods --pre --quiet
   - pod setup --silent
   - pod repo update --silent
 script:
   - pod install
-  - pod lib lint
+  - pod lib lint ParseUI.podspec
+  - pod lib lint --use-frameworks --quick ParseUI.podspec
   - xctool -workspace ParseUI.xcworkspace -scheme 'ParseUI' -sdk iphonesimulator build
   - xctool -workspace ParseUI.xcworkspace -scheme 'ParseUIDemo' -sdk iphonesimulator build

--- a/ParseUI.podspec
+++ b/ParseUI.podspec
@@ -27,6 +27,7 @@ Pod::Spec.new do |s|
                           'ParseUI/Classes/Views/*.h',
                           'ParseUI/Classes/Cells/*.h',
                           'ParseUI/Other/*.h'
+  s.xcconfig = { 'OTHER_LDFLAGS' => '$(inherited) -undefined dynamic_lookup' }
   s.frameworks          = 'Foundation',
                           'UIKit',
                           'CoreGraphics',

